addons/web/static/tests/views/fields/numeric_fields_tests.js

```diff
@@ -161,6 +161,8 @@ QUnit.module("Fields", (hooks) => {
 
             await click(target.querySelector(".o_progress"));
             const progressbarInputs = target.querySelectorAll(".o_field_progressbar input");
+            const [len1, len2] = [...progressbarInputs].map((input) => input.value.length);
+            progressbarInputs[0].setSelectionRange(len1, len1);
```

Comment:
Please explain why is this needed ?

Reply:
When clicking the progressbar, it focuses on the first input but the full value is highlighted, so I need to set the selection to the end of the value so that the "decimalPoint" is appended.

It's the thing we are fixing. Basically the original test is wrong because even if the value is highlighted, the "decimalPoint" is appended.

Reply:
Ha! I see.
While you are at it you should add a comment.
Or maybe don't do that "magic" and assert that the selection range for the progressbar field does select all.